Packs/Phishing/ReleaseNotes/3_0_3.md

```diff
@@ -0,0 +1,5 @@
+
+#### Playbooks
+##### Phishing - Generic v3
+- `DBotPredictURLPhishing` automation was added to the ML branch.
+- Added 2 playbook inputs related to the `DBotPredictURLPhishing` automation.
```

Comment:
Consider adding the names of the inputs.

Reply:
Done